wiki/Glossary/Local_Song_Offset/en.md

```diff
@@ -0,0 +1,18 @@
+# Local Song Offset
+
+*For offset while beatmapping, see: [Offset](/wiki/Beatmapping/Offset/)*
+
+**Local song offset** is an offset which moves the appearance of [hit objects](/wiki/Hit_object/) in relation to the audio of individual [beatmaps](/wiki/beatmaps). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the global offset to calculate a total offset.
```

Comment:
```suggestion
**Local song offset** is an offset which moves the appearance of [hit objects](/wiki/Hit_object) in relation to the audio of individual [beatmaps](/wiki/Beatmaps). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the global offset to calculate a total offset.
```